debian/control

```diff
@@ -23,6 +23,7 @@ Build-Depends: debhelper (>= 9),
                qml-module-io-thp-pyotherside,
                qml-module-qt-labs-settings,
                qml-module-qt-labs-platform,
+               qml-module-qt-labs-folderlistmodel,
```

Comment:
This might be overly pedantic, but let's sort these? :smile:

Reply:
That is a good idea, but will do it in a separate PR to master.